src/component/Button.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/button-has-type */
```

Comment:
I see no reason why you should disable these rules, you just need to understand them and write your code accordingly. 

- [ ] react/prop-types => If the `propType` does not have `isRequired`, ensure you add `defaultProps` value.

- [ ] react/destructuring-assignment => destructure where applicable, that's simply es6, which is a requirement for this project.

- [ ] react/button-has-type => add `type` attribute, example: `type="submit"`.

Read more about `PropTypes` [here](https://reactjs.org/docs/typechecking-with-proptypes.html).